runhouse/resources/module.py

```diff
@@ -823,6 +823,14 @@ def distribute(
                 **distribution_kwargs, name=name, replicas=replicas
             ).to(self.system, env=self.env.name)
             return pooled_module
+        elif distribution == "ray":
+            from runhouse.resources.distributed.ray_distributed import RayDistributed
```

Comment:
minor nit - for these imports we could do `from runhouse.resources.distributed import RayDistributed` and add the relevant import to the distributed package `__init__`